cp22/hpi-fun2/newsort/problem.tex

```diff
@@ -0,0 +1,28 @@
+\input{template.tex}
+
+\begin{document}
+
+\makeheader
+
+Rick and Morty are on a new adventure. This time they landed in a parallel universe and of course, Morty broke Ricks laser gun which is needed ot get back home. To get the money back for the materials for the gun they try to gamble at the stock market. Rick knows that it's a physical law, that the right stocks always go up - and thats where they want to invest. 
+
+You need to figure out if the values of a stock price you receive are increasing, so that Rick and Morty know where to invest. The problem is: In this parallel universe they use the same arabian digits but they don't have the same ordering. In our world we have the ordering $0 < 1 < 2 < 3 < 4 < 5 ...$ but in a parallel universe this ordering could be $6 < 2 < 1 < 4 \dots$ . This ordering does not apply to exponentials, so 11 is still bigger than 1.
```

Comment:
Make sure that it is clear this doesn't mean strictly increasing, but same values are allowed.